tests/unit/amo/components/TestSearch.js

```diff
@@ -159,10 +154,10 @@ describe('Search', () => {
     expect(root.find(ErrorList)).toHaveLength(1);
   });
 
-  it('should render "Enter a search term and try again." msg if no search term', () => {
-    const root = mountRender({ filters: { query: null } });
+  it('should render an error message on empty search', () => {
+    const root = render({ filters: { query: null } });
 
-    expect(root.find(Card)).toHaveText('Enter a search term and try again.');
+    expect(root.find('h1.SearchContextCard-header')).toHaveText('Enter a search term and try again.');
```

Comment:
You can just target the class without the `h1` tag; that's usually what we do and it means we can always change the tag name later 😄